From 63affa8969f858d59f5fc3efb9326172688625c0 Mon Sep 17 00:00:00 2001
From: Mahmoud Al-Qudsi <mqudsi@neosmart.net>
Date: Mon, 23 Dec 2024 14:29:00 -0600
Subject: [PATCH 2/2] Fix typo in hard-coded name of SIGSTKFLT

---
 src/signal.rs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/signal.rs b/src/signal.rs
index 699ff455c..26f4b19c2 100644
--- a/src/signal.rs
+++ b/src/signal.rs
@@ -402,7 +402,7 @@ const SIGNAL_TABLE : &[LookupEntry] = &[
     LookupEntry::new(libc::SIGINFO,   L!("SIGINFO"), L!("Information request")),
 
     #[cfg(all(target_os = "linux", not(target_arch = "mips64")))]
-    LookupEntry::new(libc::SIGSTKFLT, L!("SISTKFLT"), L!("Stack fault")),
+    LookupEntry::new(libc::SIGSTKFLT, L!("SIGSTKFLT"), L!("Stack fault")),
 
     #[cfg(target_os = "linux")]
     LookupEntry::new(libc::SIGIOT,   L!("SIGIOT"), L!("Abort (Alias for SIGABRT)")),
-- 
2.48.1

